src/bezpath.rs

```diff
@@ -440,6 +440,63 @@ impl BezPath {
         }
         cbox.unwrap_or_default()
     }
+
+    /// Returns a new path with the winding direction of all subpaths reversed.
+    ///
+    /// If the path does not start with a move, returns `None`.
```

Comment:
The behavior doesn't quite match the comment - on empty path input, the result is also empty (which is valid), but it's not a path that starts with a move. Given that a path starting with an element other than move is an invalid state, I'm not sure it should complicate the return type.

Reply:
My initial cut didn't have the Option but I was trying to match behavior of the original code. On second thought, we should be doing validation outside of contour reversal, so I'll go ahead and remove this.